config/migrate_plus.migration.report_files.yml

```diff
@@ -41,13 +41,13 @@ process:
     plugin: format_date
     from_format: Y-m-d
     to_format: Y/m/
-    source: StartDate
+    source: EndDate
```

Comment:
This was set to EndDate on the presumption that reports arrive at the end of a period, and so we'd want them in the folder that corresponds to that.